### PR TITLE
Monitor limit increase

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

https://sumologic.atlassian.net/browse/SUMO-225690

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]

File: docs/alerts/monitors/monitor-faq.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ deployment=acme region=us-east1 metric=container_cpu_usage_seconds_total | rate
 
 And so on.
 
+## Is there a limit to the number of monitors I can have?
+
+You can have up to 5,000 monitors (active and inactive) simultaneously.
```
Comment (rmamilla-sumo) [inline docs/alerts/monitors/monitor-faq.md:53]: By default the limit is per the subscription type. user need to reach out to customer support to bump to 5k monitors. i think we should mention that.

Comment (kimsauce) [reply docs/alerts/monitors/monitor-faq.md:53]: Thanks for the quick feedback! Where can the user find their current subscription type limit? I don't see it under **Account** > **Administration**.

Comment (rmamilla-sumo) [reply docs/alerts/monitors/monitor-faq.md:53]: documented @ https://docs.google.com/document/d/18GOB3Ppo7wa1ft-RY1ePk1ymgT0x43AuAy1rhG3Nf6k/edit# but don't know if we want to do @ docs

Comment (rmamilla-sumo) [reply docs/alerts/monitors/monitor-faq.md:53]: @kimsauce , 
can we say , you can increase the monitor limit up to 5000 by reaching out to customer support?

Comment (kimsauce) [inline docs/alerts/monitors/monitor-faq.md:53]:
```suggestion
See [Sumo Logic Pricing](https://www.sumologic.com/pricing) to learn the number of monitors (active and inactive) you can run simultaneously. You can increase the monitor limit up to 5000 by reaching out to our support team.
```

Comment (kimsauce) [reply docs/alerts/monitors/monitor-faq.md:53]: @rmamilla-sumo , thank you for your help! What do you think of these edits?

Comment (rmamilla-sumo) [reply docs/alerts/monitors/monitor-faq.md:53]: Looks good. Thanks.

**Review by rmamilla-sumo — APPROVED** [review]
LGTM

**Pull request merged**